docs/java/how-to/logging-overview.md

```diff
@@ -0,0 +1,200 @@
+---
+id: logging-overview
+title: Logging with the SAP Cloud SDK for Java
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Logging Overview
+description: This document will give you an overview of the different ways to log information with your application.
+---
+
+This document aims to give you an overview of how you can write events of your application with the standard logging frameworks and how the SAP Cloud SDK integrates with that.
+
+## Application Logging
+
+To better differentiate between different kinds of logging we will call the type of logs that are usually only relevant for developers to understand why the system behaves the way it did (for example during debugging) as _Application Logging_.
+Other types of logs might be kept due to legislative requirements (audit trails or audit logging) or be part of the terms and conditions (business logging).
+
+[//]: # (Add some points of the history of logging in Java)
+
+[//]: # (Add a section on what SLF4J is)
+
+### Write log messages in your application
+
+To start writing log messages you need two dependencies:
+
+- The SLF4J API `org.slf4j:slf4j-api`
+- A logging implementation of your choice (e.g. logback `ch.qos.logback:logback-classic`)
+
+Depending on the chosen logging implementation you might not _need_ to specify the SLF4J API, but it's in general best practice to not rely on transitive dependencies and therefore reference the SLF4J API anyway.
+
+Also, if you deploy your application to SCP CloudFoundry using the SAP Java Buildpack you get the logging implementation provided at runtime by the container. This means, if you only run the application on Cloud Foundry you don't need to provide any implementation.
+However, if you want to locally run your application (e.g. with TomEE with `mvn tomee:run`) you will need a logging implementation with at least the maven scope `runtime`.
```

Comment:
Man! This deserves O'Reilly or Addison Weisely publishing))) Quite an introduction!